src/components/overview/ImageGallery.jsx

```diff
@@ -1,11 +1,58 @@
-import React from 'react';
+import React, {useState} from 'react';
 import styles from './overview.module.css'
+import json from './styleSample.json';
 
 
 const ImageGallery = (props) => {
 
+  var nums = Array.from(Array(7).keys()) //Used for testing flex box children size
+  // console.log(nums)
```

Comment:
Will you still be needing this console log?

Reply:
Removed comment.